Sheetscripts/CleanUp/HTML.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
+  </head>
+  <body id="bodyyy">
+    
+    <div id="rawFilePrompt">
+     <label for="firstname">Raw Student Data Sheet:</label>
```

Comment:
Should change all of these labels to indicate what they are actually for, or is there a reason they all say for "firstname"?